src/sparsezoo/utils/registry.py

```diff
@@ -194,19 +239,24 @@ def get_from_registry(
     :return: value from retrieved the registry for the given name, raises
         error if not found
     """
+    name = standardize_lookup_name(name)
 
     if ":" in name:
         # user specifying specific module to load and value to import
         module_path, value_name = name.split(":")
         retrieved_value = _import_and_get_value_from_module(module_path, value_name)
     else:
+        # look up name in alias registry
+        name = _ALIAS_REGISTRY[parent_class].get(name)
```

Comment:
actually don't we need to standardize aliases? Because on line 242 we're standardizing the input, so we wouldn't match to an unstandardized alias